docs/installation/native-installation.md

```diff
@@ -45,15 +45,28 @@ php phing build-demo-dev
 
 ## Run integrated HTTP server
 
-```sh
-php bin/console server:run
+First follow [installation instructions](https://symfony.com/doc/current/setup/symfony_server.html#installation) for the `symfony` binary which provides its own web server.
+
+Then you can run integrated webserver for the first domain:
+```bash
+symfony server:start
```

Comment:
If you do not want to paste phing target here.... would not be better to use php bin/console server:start as we use it in other docs?

Reply:
`php bin/console server:start` was provided by symfony/webserver-bundle and this command no longer exists. Instead, the Symfony command is used. 
Although it's possible to create a command that calls `symfony server:run` internally I'm not a big fan of that solution because it counts on executable available on the machine and in the end it's just a different name and I think symfony executable provides many great features and it can be counterproductive to hide it behind our command.

_(yes, the same can be said of the phing target and I seriously consider removing that target completely)_

I searched for usages of `php bin/console server:start` in the documentation, but didn't find any. Do you have an example, please?

Reply:
I meant, that we do not use commands starting with symfony. I just expectted it is alias for php bin/console, which we use...